File: 3-Bimestre/C++/ex-7.cpp
#include <iostream>
using namespace std;

int main() {
    int n, contador = 1;
    int positivos = 0, negativos = 0, zeros = 0;

    while (contador <= 10) {
        cout << "Digite um numero: ";
        cin >> n;

        if (n > 0)
            positivos++;
        else if (n < 0)
            negativos++;
        else
            zeros++;

        contador++;
    }

    cout << "Positivos: " << positivos << endl;
    cout << "Negativos: " << negativos << endl;
    cout << "Zeros: " << zeros;

    return 0;
}

File: 3-Bimestre/Turma-A/ex-7.py
contador = 1
positivos = 0
negativos = 0
zeros = 0

while contador <= 10:
    n = int(input(f'Digite o {contador}º número: '))
    if n > 0:
        positivos += 1
    elif n < 0:
        negativos += 1
    else:
        zeros += 1
    contador += 1

print(f'Quantidade de positivos: {positivos}')
print(f'Quantidade de negativos: {negativos}')
print(f'Quantidade de zeros: {zeros}')
